tools/dfpl/train/dfpl-train.xml

```diff
@@ -0,0 +1,230 @@
+                       help="Track FNN performance via Weights &amp; Biases."/>
+                <param label="Track Autoencoder" argument="--aeWabTracking"
+                       type="boolean"
+                       help="Track autoencoder performance via Weights &amp; Biases."/>
+            </section>
+        </section>
+    </inputs>
+    <outputs>
+        <!--        todo: filter -> let user decide if they want output svg/csv or nothing -->
+        <!--        <data name="loss_table" format="csv" from_work_dir="output/" label="${tool.name} on ${on_string}: csv">-->
+        <!--        </data>-->
+        <!--        <data name="loss_diagram" format="svg" from_work_dir="output/" label="${tool.name} on ${on_string}: svg">-->
+        <!--        </data>-->
+        <!--        todo: switch to hdf5-->
+        <data name="output_model_weights" label="${tool.name} on ${on_string}: model weights"
+              format="zip"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="inputFile" value="S_dataset.csv"/>
+            <param name="verbose" value="2"/>
+            <param name="trainAC" value="true"/>
+            <param name="compressFeatures" value="true"/>
+            <param name="encFPSize" value="256"/>
+            <param name="aeSplitType" value="random"/>
+            <param name="aeEpochs" value="2"/>
+            <param name="aeBatchSize" value="351"/>
+            <param name="aeActivationFunction" value="relu"/>
+            <param name="aeLearningRate" value="0.001"/>
+            <param name="aeLearningRateDecay" value="0.0001"/>
+            <param name="aeType" value="deterministic"/>
+            <param name="type" value="smiles"/>
+            <param name="fpType" value="topological"/>
+            <param name="fpSize" value="2048"/>
+            <param name="split_type" value="random"/>
+            <param name="enableMultiLabel" value="false"/>
+            <param name="sampleFractionOnes" value="0"/>
+            <param name="sampleDown" value="false"/>
+            <param name="trainFNN" value="true"/>
+            <param name="kFolds" value="1"/>
+            <param name="threshold" value="0.5"/>
+            <param name="testSize" value="0.2"/>
+            <param name="fnnType" value="FNN"/>
+            <param name="optimizer" value="Adam"/>
+            <param name="lossFunction" value="bce"/>
+            <param name="epochs" value="11"/>
+            <param name="batchSize" value="128"/>
+            <param name="activationFunction" value="selu"/>
+            <param name="dropout" value="0.0107"/>
+            <param name="learningRate" value="2.2e-06"/>
+            <param name="l2reg" value="0.001"/>
+            <param name="aeWabTracking" value="false"/>
+            <param name="wabTracking" value="false"/>
+            <!--            todo: add tests for svg, csv -->
+        </test>
+    </tests>
+    <help><![CDATA[
+    TODO: Fill in help.
+    ]]></help>
```

Comment:
Also add citation.

Reply:
Added the citations now. Help text will come soon